src/core_plugins/metrics/public/kbn_vis_types/editor_controller.js

```diff
@@ -1,6 +1,6 @@
 import modules from 'ui/modules';
 import '../services/executor';
-import 'plugins/timelion/directives/refresh_hack';
+import '../../../timelion/public/directives/refresh_hack';
```

Comment:
Just a quick look through the code and it looks like if we added `$rootScope.$broadcast('fetch');` to https://github.com/elastic/kibana/blob/master/src/ui/public/courier/courier.js#L69-L78 and we can get rid of `refresh-hacks` all together.